### PR TITLE
chore: make `@playwright/test` depend on `playwright`

### PR DESCRIPTION
**Pull request by dgozman**

### CODE REVIEW

**Review by pavelfeldman — APPROVED** [review]

File: tests/components/ct-vue-cli/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "core-js": "^3.8.3",
-    "vue": "^3.2.13",
+    "vue": "3.2.36",
```
Comment (mxschmitt) [inline tests/components/ct-vue-cli/package.json:13]: this looks unrelated? We kept it unpinned to see when the world is broken.

File: tests/installation/fixture-scripts/esm-playwright-test.mjs [hunk 2]
```diff
@@ -19,6 +19,9 @@ import * as playwright from '@playwright/test';
 import defaultExport from '@playwright/test';
 import testESM from './esm.mjs';
 
+console.log('test:', test);
```
Comment (mxschmitt) [inline tests/installation/fixture-scripts/esm-playwright-test.mjs:22]: this looks unrelated

**Review by mxschmitt — APPROVED** [review]

File: tests/components/ct-vue-cli/package.json (hunk 1 above)
Comment (dgozman) [reply tests/components/ct-vue-cli/package.json:13]: Since this PR regenerated package.json, we ran into conflicts right away.
